src/iOS/toga_iOS/window.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, view):
     def statusbar_height(self):
         if UIApplication.sharedApplication.statusBarOrientation == UIInterfaceOrientation.Portrait.value:
```

Comment:
At column 12: (F405) 'UIApplication' may be undefined, or defined from star imports: .libs